MyLibs/fishAnalysis.cpp

```diff
@@ -211,6 +211,17 @@ bool FishData::findHeadSide(Point2f* M)
 	return areas[aIdx] > areas[!aIdx];
 }
 
+void ArenaData::prepareBgImg(int width,int height,int cIdx, Pylon::CPylonImage pylonImg) {
+	Mat rawImg = Mat(width, height,
+		CV_8UC1, (uint8_t*)pylonImg.GetBuffer());
```

Comment:
Just transfer the converted image with the data type ``(uint8_t*)``. It would be fine.